pkg/metrics/metrics.jsx

```diff
@@ -302,6 +309,17 @@ class CurrentMetrics extends React.Component {
     componentDidMount() {
         superuser.addEventListener("changed", () => this.setState({ podNameMapping: {} }));
         cockpit.user().then(user => this.setState({ userid: user.id }));
+        this.networkModel = new NetworkManagerModel();
```

Comment:
Does this properly react to restarting NM, or NM not running at all? (Think Ubuntu or embedded with only using systemd-networkd)

Reply:
Fair, again, I agree that always showing a link and fixing it elsewhere sounds wiser, I just tried to understand if it would work

Reply:
So properly answering this, I grabbed the test image and stripped network manager from it, system cockpit-network requires network manager to function, if you try to load `127.0.0.2:9291/network#/cockpittest1`, you get greeted by:

![image](https://github.com/cockpit-project/cockpit/assets/1463740/8de98c05-f952-4b08-adb4-d47ed1eb06b7)

Do you think it is worth getting rid of the link altogether in that case?